packages/core/src/external/fhir/bundle/qa.ts

```diff
@@ -2,41 +2,43 @@ import { Bundle, Resource } from "@medplum/fhirtypes";
 import { MetriportError } from "@metriport/shared";
 import { filterTruthy } from "@metriport/shared/common/filter-map";
 import { uniq } from "lodash";
-import { out } from "../../../util";
+import { capture, out } from "../../../util";
+import { getPatientReferencesFromResources, isPatientReference } from "../patient/reference";
+import { isPatient } from "../shared";
+import { getIdFromReference } from "../shared/references";
 import { getPatientsFromBundle } from "./patient";
 
 /**
  * Check the FHIR bundle only contains references for the given patient.
  * @param bundle
  * @param cxId
  * @param patientId
- * @returns true if the bundle is valid, otherwise throws an error
+ * @returns true if the bundle is valid, otherwise throws an error indicating the error
  */
-export function checkBundleForPatient(
-  bundle: Bundle<Resource>,
-  cxId: string,
-  patientId: string
-): true {
-  const { log } = out(`checkBundleForPatient - cx ${cxId}, pat ${patientId}`);
+export function checkBundle(bundle: Bundle<Resource>, cxId: string, patientId: string): true {
+  const { log } = out(`checkBundle - cx ${cxId}, pat ${patientId}`);
```

Comment:
never seen `pat` in logging

Reply:
trying to save real estate on the logs, sometimes its almost impossible to read the important stuff because of too much repeated content.